tracer/build/_build/Build.cs

```diff
@@ -201,6 +201,15 @@ void DeleteReparsePoints(string path)
         .DependsOn(CompileIntegrationTests)
         .DependsOn(BuildRunnerTool);
 
+    Target BuildWindowsDebuggerIntegrationTests => _ => _
+        .Unlisted()
+        .Requires(() => IsWin)
+        .Description("Builds the debugger integration tests for Windows")
+        .DependsOn(CompileManagedTestHelpers)
+        .DependsOn(CreatePlatformlessSymlinks)
+        .DependsOn(CompileDebuggerIntegrationTests)
+        .DependsOn(BuildRunnerTool);
```

Comment:
Do you need to build the runner tool?

Reply:
FWIW I'm personally not aware of any reason we would need to.

Reply:
Removed at 59340936a420da68670139fcd116e6b479f1c1d0